google/data_source_dns_managed_zone_test.go

```diff
@@ -0,0 +1,70 @@
+package google
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"testing"
+)
+
+func TestAccDataSourceDnsManagedZone_basic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckDnsManagedZoneDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccDataSourceDnsManagedZone_basic,
+				Check:  testAccDataSourceDnsManagedZoneCheck("qa", "foo"),
+			},
+		},
+	})
+}
+
+func testAccDataSourceDnsManagedZoneCheck(dsName, rsName string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		dsFullName := "data.google_dns_managed_zone." + dsName
+		rsFullName := "google_dns_managed_zone." + rsName
+		ds, ok := s.RootModule().Resources[dsFullName]
+		if !ok {
+			return fmt.Errorf("root module has no resource called %s", dsFullName)
+		}
+
+		rs, ok := s.RootModule().Resources[rsFullName]
+		if !ok {
+			return fmt.Errorf("can't find %s in state", rsFullName)
```

Comment:
Other than that I copied and pasted this snippet from another test. No ;) 

Fixed